core/src/components/chip/test/shape/chip.e2e.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  /**
+   * This behavior only applies to Ionic Theme.
```

Comment:
We should reference the tech debt ticket here too. When this gets added back to the iOS/MD themes these tests will apply to those themes as well and will need to be updated.

Reply:
I added a code suggestion to get this added 🙂